api/src/controllers/anime/createAnime.ts

```diff
@@ -0,0 +1,76 @@
+import type { Request, Response, NextFunction } from '@@types/express.js';
+import { errors, entities } from '@@utils/index.js';
+import { Anime } from '@@entities/index.js';
+
+interface Params {
+    id: number
+};
+
+interface Body {
+        // Define the expected keys and their types here
+        title?: string;
+        description?: string;
+        episodes?: number;
+        rating?: number;
+        status?: string;
+    }
+
+// Create
+async function create(req: Request<Body>, res: Response<Params>, next: NextFunction) {
```

Comment:
This function should just house the logic for creating a new Anime entity, but it seems like you're performing multiple CRUD operations